using-timescaledb/continuous-aggregates.md

```diff
@@ -459,23 +412,25 @@ hypertable. For instance, we may want to set the materialization hypertable's
 accomplished by running [`set_chunk_time_interval`][api-set-chunk-interval] on
 the materialization hypertable.  
 
-**Creating Indexes on the Materialization Hypertable:**
-By default, the database will automatically create composite indexes on each
-column specified in the GROUP BY combined with the time_bucket column (i.e. in
-our example, we would create an index on `{device_id, bucket}` and if we had
-grouped by other columns, `foo` and `bar`, we would also create indexes on
-`{foo, bucket}` and `{bar, bucket}`). Setting `timescaledb.create_group_indexes`
-to `false` when creating the view will prevent this.  If we want to create
-additional indexes or drop some of the default ones, we can do so by creating or
-dropping the appropriate indexes on the materialization hypertable directly.
+**Creating Indexes on the Materialization Hypertable:** By default,
+the database will automatically create composite indexes on each
+column specified in the `GROUP BY` combined with the `time_bucket`
+column (i.e. in our example, we would create an index on `{device_id,
+bucket}` and if we had grouped by other columns, `foo` and `bar`, we
+would also create indexes on `{foo, bucket}` and `{bar,
+bucket}`). Setting `timescaledb.create_group_indexes` to `false` when
+creating the view will prevent this.  If we want to create additional
+indexes or drop some of the default ones, we can do so by creating or
+dropping the appropriate indexes on the materialization hypertable
+directly.
```

Comment:
Add that the name of the materialization table is found in the `continuous_aggregates` view?